src/bin/convert_from_json.rs

```diff
@@ -0,0 +1,208 @@
+                writer.unsigned_long_at(val, &path).expect("Error");
+            }
+            binjs::io::binjs_json::read::NextType::Bool => {
+                let path = reader.get_path();
+                let val = reader.bool_at(&path).expect("Error");
+                writer.bool_at(val, &path).expect("Error");
+            }
+            binjs::io::binjs_json::read::NextType::List => {
+                let path = reader.get_path();
+                let len = reader.enter_list_at(&path).expect("Error");
+                writer.enter_list_at(len as usize, &path).expect("Error");
+            }
+            binjs::io::binjs_json::read::NextType::EndOfList => {
+                let path = reader.get_path();
+                reader.exit_list_at(&path).expect("Error");
+                writer.exit_list_at(&path).expect("Error");
+            }
+            binjs::io::binjs_json::read::NextType::TaggedTuple => {
+                let path = reader.get_path();
+                let (name, maybe_field_names) = reader.enter_tagged_tuple_at(&path).expect("Error");
+                let field_names = maybe_field_names.expect("Field names should be populated");
+
+                let mut children = Vec::new();
+                for name in field_names.iter() {
+                    children.push(name);
+                }
+
+                let node = DummyNode::new();
+                writer
+                    .enter_tagged_tuple_at(&node, &name, children.as_slice(), &path)
+                    .expect("Error");
+            }
+            binjs::io::binjs_json::read::NextType::EndOfTaggedTuple => {
+                let (name, field_names) = reader.current_tagged_tuple_info();
+
+                let path = reader.get_path();
+                reader.exit_tagged_tuple_at(&path).expect("Error");
+
+                let mut children = Vec::new();
+                for name in field_names.iter() {
+                    children.push(name);
+                }
+
+                let node = DummyNode::new();
+                writer
+                    .exit_tagged_tuple_at(&node, &name, children.as_slice(), &path)
+                    .expect("Error");
+            }
+            binjs::io::binjs_json::read::NextType::End => {
+                break;
+            }
+        }
+    }
+}
+
+fn main() {
+    thread::Builder::new()
+        .name("large stack dedicated thread".to_string())
+        .stack_size(20 * 1024 * 1024)
+        .spawn(|| {
+            main_aux();
+        })
+        .expect("Could not launch dedicated thread")
+        .join()
+        .expect("Error in dedicated thread");
+}
+
+fn main_aux() {
+    env_logger::init();
+
+    let matches = App::new("Converter from BinAST JSON format to other BinAST format")
+        .author("arai <arai@mozilla.com>")
+        .about("Convert a BinAST JSON file to other BinAST format file. The JSON file is exported by binjs_encode with advanced json option")
```

Comment:
Maybe "To obtain a BinAST JSON file, you may use binjs_encode with command `advanced json`."